content/en/docs/components/pipelines/v2/quickstart.md

```diff
@@ -245,22 +269,31 @@ url = f'{endpoint}/#/runs/details/{run.run_id}'
 
-Within the scope of a component, artifacts can be read (for inputs) and written (for outputs) via the `.path` attribute. The KFP backend will ensure that *input* artifact files are copied *to* the executing pod local filesystem from remote storage at runtime so that the component function can read input artifacts from the local filesystem. By comparison, *output* artifact files are copied *from* the pod's local filesystem to remote storage when the component finishes running, this way output artifacts are persisted outside of the pod. In both cases, the component author only needs to interact with the local filesystem to create persistent artifacts.
+  The arguments for the parameters annotated with `Output[]` are not passed to components by the pipeline author. The KFP backend passes this artifact during component runtime, so that component authors don't need to be concerned about the path to which the output artifacts are written. After an output artifact is written, the backend executing the component recognizes the KFP artifact types (`Dataset` or `Model`), and organizes them on the Dashboard.
 
-Notice that arguments for the parameters annotated with `Output[]` are not passed to components by the pipeline author. The KFP backend passes this artifact at component runtime so that component authors do not need to be concerned with the path to which output artifacts should be written. Once an output is written, the executing backend will recognize the KFP artifact types (`Dataset`, `Model`, etc.) and organize them within the Dashboard.
+  An output artifact can be passed as an input to a downstream component using `.outputs` attribute of the source task and the output artifact parameter name, as follows:
+  
+  `create_dataset_task.outputs['iris_dataset']`
 
-An output artifact can be passed as an input to a downstream component using via the source task's `.outputs` attribute and the output artifact parameter name: `create_dataset_task.outputs['iris_dataset']`.
+* One of the **DSL control flow features**, `dsl.ParallelFor`, is used. It is a context manager that lets pipeline authors write a `for` loop. This `for` loop is executed at pipeline runtime. Using `dsl.ParallelFor` to iterate over the `neighbors` pipeline argument lets you execute the  `train_model` component with different arguments and test multiple hyperparameters in one pipeline run. Other control flow features include `dsl.Condition` and `dsl.ExitHandler`.
 
-Third, we used one of the **DSL control flow features**. `dsl.ParallelFor` is a context manager that allows pipeline authors to write a for loop that is executed at pipeline runtime. By using `dsl.ParallelFor` to iterate over the `neighbors` pipeline argument, we can execute the  `train_model` component with different arguments to test multiple hyperparameters in one pipeline run. Other control flow features include `dsl.Condition` and `dsl.ExitHandler`.
+## Congratulations!
```

Comment:
Changing this to a heading style might make the ending look more prominent, but is not mandatory.